dapps/dapps_dev.json

```diff
@@ -612,14 +564,6 @@
         "community"
       ]
     },
-    {
-      "name": "Tinkernet Rewards Claim",
```

Comment:
Let's also remain this one, it's relatively new

Reply:
fixed